### ISSUE TITLE
Pikemen Shoulder Shield

### ISSUE THREAD

**Issue by StanleySweet** [0ADMods/kushites#13]
Pikemen should wear their shield on their shoulder.

**Comment by JustusAvramenko** [comment]
Yes. The current small round shields do not sit properly on the arm. There needs to be a couple new meshes exported the same angle and position as the Macedonian-style pikeman shield. Or are they going to be positioned higher up on these guys?
